### PR TITLE
[BD-26][EDUCATOR-5843] doc: Add mfe special exam lib local development instructions

### PR DESCRIPTION
**Pull request by idegtiarov**
**Description:**

Update `developing` documentation with the frontend special exam library setup for the local development.

**JIRA:**

[EDUCATOR-5843](https://openedx.atlassian.net/browse/EDUCATOR-5843)

**Pre-Merge Checklist:**

- [ ] Updated the version number in `edx_proctoring/__init__.py` and `package.json` if these changes are to be released.
- [x] Described your changes in `CHANGELOG.rst`
- [x] Confirmed Github reports all automated tests/checks are passing.
- [ ] Approved by at least one additional reviewer.

**Post-Merge:**

- [ ] Create a tag matching the new version number.

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @idegtiarov! I've created [BLENDED-893](https://openedx.atlassian.net/browse/BLENDED-893) to keep track of it in Jira.   More details are on the [BD-26](https://openedx.atlassian.net/wiki/spaces/COMM/pages/2523627980/BD-26+Proctoring+MFE+and+configuration) project page.


When this pull request is ready, tag your edX technical lead.

File: docs/developing.rst [hunk 1]
```diff
@@ -54,6 +54,42 @@ In edx-platform/lms/envs/private.py and edx-platform/cms/envs/private.py:
...
+
+How do I setup `mfe-special-exam-lib` for local development?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+`MFE special exam lib <https://github.com/edx/frontend-lib-special-exams/>`_ is a react library to support
+special exams in the MFE application learning.
+
+Make sure that `frontend-app-learning` is setup and running on your devstack.
+
+The special exam lib is installed as a dependency of MFE application learning.
```
Comment (schenedx) [inline docs/developing.rst:66]: #NIT `The special exam lib is installed as a dependency of MFE application learning.` should be `The special exam lib is installed as a dependency of Learning MFE.`

File: docs/developing.rst [hunk 2]
```diff
@@ -54,6 +54,42 @@ In edx-platform/lms/envs/private.py and edx-platform/cms/envs/private.py:
         'MUST_BE_VERIFIED_TRACK': False
     }
 
+
+How do I setup `mfe-special-exam-lib` for local development?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+`MFE special exam lib <https://github.com/edx/frontend-lib-special-exams/>`_ is a react library to support
+special exams in the MFE application learning.
```
Comment (schenedx) [inline docs/developing.rst:62]: s/`MFE application learning`/`Learning MFE`

**Review by schenedx — APPROVED** [review]
👍 Just NITs. Thank you for the instructions!

File: docs/developing.rst (hunk 1 above)
Comment (idegtiarov) [reply docs/developing.rst:66]: fixed

File: docs/developing.rst (hunk 2 above)
Comment (idegtiarov) [reply docs/developing.rst:62]: done

**Comment by idegtiarov** [comment]
Thanks for the review, Simon!

**Comment by Zacharis278** [comment]
changes to developing file look good. Can we also update the system-overview doc to include the new frontend?

**Comment by idegtiarov** [comment]
@Zacharis278 nice point will propose updates to the system-overview doc as well.

**Pull request merged**